src/main/java/com/banking/money/transaction/AccountsDaoImpl.java

```diff
@@ -11,7 +11,7 @@
 import com.banking.exceptions.AccountsDBAccessException;
 
 /**
- * @author Dipanjan Sengupta
+ * @author Dipanjan Sengupta 
  * @purpose - HibernateTemplate for operations on accounts table
  */
 public class AccountsDaoImpl implements AccountsDao{
```

Comment:
- hibernate template shouldn't have a getter method since it is dependency injected
- log all TransactionException after catching them and before wrapping into custom exception
- readOnly property of Transactional annotation for get all account, get account with customer id should be true